scripts/prepare_lambda_deployment.py

```diff
@@ -70,6 +77,56 @@ def patch_cloudwatch_resource(resource_type_name, property_name):
     return input_json
 
 
+def patch_resource_names(input_json: JSON) -> JSON:
```

Comment:
`input` is redundant. Every argument is usually an input. This is a specific type of JSON and the name should reflect that.

Reply:
Should I rename the `input_json` argument in `transform_tf` (https://github.com/DataBiosphere/azul/pull/1701/files#diff-869f5860e107030dfbec5ee431581404R32) as well since I'm working in this file anyway?